shared/chat/conversation/input-area/suggestors/users.tsx

```diff
@@ -166,7 +166,9 @@ const useDataTeams = () => {
     () =>
       inboxLayout?.bigTeams?.reduce<Array<TeamListItem>>((arr, t) => {
         if (t.state === T.RPCChat.UIInboxBigTeamRowTyp.channel) {
-          arr.push({channelname: t.channel.channelname, teamname: t.channel.teamname})
+          if (t.channel.channelname.length) {
```

Comment:
this should practically never happen as we try and ensure a channel name but i happen to have a blank channel name in a team and this will cause a key dupe issue so might as well sanity check here